src/industry_cmd.cpp

```diff
@@ -412,6 +412,7 @@ static void AddAcceptedCargo_Industry(TileIndex tile, CargoArray &acceptance, Ca
 {
 	IndustryGfx gfx = GetIndustryGfx(tile);
 	const IndustryTileSpec *itspec = GetIndustryTileSpec(gfx);
+	const IndustrySpec *indspec = GetIndustrySpec(GetIndustryType(tile));
```

Comment:
Bikeshed: "ind" is already queried, so "ind->type" gives the same as "GetIndustryType(tile)".